.gitignore

```diff
@@ -3,8 +3,10 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is included in the `.gitignore` file. If user-uploaded files need to be tracked, consider removing this line. Otherwise, this is correct if these files should not be tracked.